arviz/tests/external_tests/test_data_beanmachine.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=no-member, invalid-name, redefined-outer-name
+import numpy as np
+import pytest
+
+from ...data.io_beanmachine import from_beanmachine  # pylint: disable=wrong-import-position
+from ..helpers import (  # pylint: disable=unused-import, wrong-import-position
+    chains,
+    check_multiple_attrs,
```

Comment:
```suggestion
```

not breaking pylint because of the ignore needed for the fixture IIRC, but not used, so better to remove it